src/Microsoft.Identity.Client/ClientApplicationBase.WithBuilders.cs

```diff
@@ -106,6 +107,32 @@ internal ClientApplicationBase(ApplicationConfiguration config)
             return await handler.RunAsync(CancellationToken.None).ConfigureAwait(false);
         }
 
+        async Task<Uri> IClientApplicationBaseExecutor.ExecuteAsync(
+            AcquireTokenCommonParameters commonParameters,
+            GetAuthorizationRequestUrlParameters authorizationRequestUrlParameters,
+            CancellationToken cancellationToken)
+        {
+            var requestParameters = CreateRequestParameters(commonParameters, UserTokenCacheInternal);
+            requestParameters.Account = authorizationRequestUrlParameters.Account;
+            requestParameters.LoginHint = authorizationRequestUrlParameters.LoginHint;
+
+            if (!string.IsNullOrWhiteSpace(authorizationRequestUrlParameters.RedirectUri))
+            {
+                // TODO(migration): should we wire up redirect uri override across the board and put this in the CreateRequestParameters method?
+                requestParameters.RedirectUri = new Uri(authorizationRequestUrlParameters.RedirectUri);
+            }
+
+            var handler = new InteractiveRequest(
+                ServiceBundle,
+                requestParameters,
+                authorizationRequestUrlParameters.ToInteractiveParameters(),
+                null);
+
+            // todo: need to pass through cancellation token here
+            return await handler.CreateAuthorizationUriAsync().ConfigureAwait(false);
+        }
+
+
         /// <summary>
         /// [V3 API] Attempts to acquire an access token for the <paramref name="account"/> from the user token cache, 
         /// with advanced parameters controlling the network call. See https://aka.ms/msal-net-acquiretokensilent for more details
```

Comment:
in the XML comment for account, we need to say that the parameter is optional and that if nothing is passed, and no .WithAccount or .WithLoginHint is provided:
- if there is one and only one account in the cache, it's picked up
- otherwise an exception is thrown